cassandra/src/main/java/org/hibernate/ogm/datastore/cassandra/impl/configuration/CassandraConfiguration.java

```diff
@@ -15,7 +15,7 @@
  *
  * @author Jonathan Halliday
  */
-public class CassandraConfiguration extends DocumentStoreConfiguration {
```

Comment:
I'd rather see such changes extracted as a different PR, which we could already apply?